include/mbgl/util/geo.hpp

```diff
@@ -21,17 +22,29 @@ using ScreenBox        = mapbox::geometry::box<double>;
 
 class LatLng {
 public:
-    struct null {};
-
     double latitude;
```

Comment:
Can we mark `latitude` and `longitude` as `const` here? We validate their values when initializing `LatLng`, but these are free to be modified after without checks.

Another option is to move these checks to a setter, and having these private.